modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -25,6 +25,13 @@ export default ({fields}) => {
 			setItemSelected(window.location.hash.match(/.*(\d+)$/)[1]);
 		}
 	}
+	else if (document.getElementById(window.location.hash.split('#')[1])) {
+		const element = document.getElementById(
+			window.location.hash.split('#')[1]
+		);
+		element.scrollIntoView();
+	}
```

Comment:
@alinedoleron, why we don't move this logic to the `useState` initialization? Please check https://reactjs.org/docs/hooks-reference.html#lazy-initial-state.